src/tokens/lst_tokens/token_lst_add_back.c

```diff
@@ -3,15 +3,24 @@
 /*                                                        :::      ::::::::   */
 /*   token_lst_add_back.c                               :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: jodufour <jodufour@student.42.fr>          +#+  +:+       +#+        */
+/*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/03/08 19:43:54 by mcourtoi          #+#    #+#             */
-/*   Updated: 2023/03/31 07:08:25 by jodufour         ###   ########.fr       */
+/*   Updated: 2023/04/03 17:56:28 by mcourtoi         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
+/**
+ * @brief Add a new token with the given parameters at the end of list.
+ * 		If list is empty, assign the new token to head.
+ * 
+ * @param list the token_lst we add to.
```

Comment:
As it was previously said, `token_lst` is rather a variable/constant name, and should not be used this way in a parameter description. We would rather say `The token list to add the new node to`.